Doc/library/functools.rst

```diff
@@ -347,8 +347,7 @@ The :mod:`functools` module defines the following functions:
 
       def partial(func, /, *args, **keywords):
           def newfunc(*more_args, **more_keywords):
-              keywords_union = {**keywords, **more_keywords}
-              return func(*args, *more_args, **keywords_union)
+              return func(*args, *more_args, **keywords | more_keywords)
```

Comment:
```suggestion
              return func(*args, *more_args, **(keywords | more_keywords))
```
Optional, but I feel it's clearer to parenthesize these to make the order of operations clear.

If you disagree, feel free to merge without the parentheses.